listen.py

```diff
@@ -43,6 +43,41 @@ def from_json(cls, j):
             data=j.get('track_metadata')
         )
 
+    @classmethod
+    def from_influx(cls, row):
+        """ Factory to make Listen objects from an influx row
+        """
+        dt = datetime.strptime(row['time'] , "%Y-%m-%dT%H:%M:%SZ")
+        t = int(dt.strftime('%s'))
+        mbids = []
+        if row.get('artist_mbids'):
```

Comment:
Right. That definitely sounds like something that might happen. Let's make sure that we get tests written for this as soon as possible, then. For now how about we change this part to my first suggestion (use `.get()` into a variable and then use the variable)?